### PR TITLE
Add startup event log for ANCM inproc

### PR DESCRIPTION
**Pull request by jkotalik**
For #982 . Can't effectively add tests until we have container infra setup.

### CODE REVIEW

**Comment by pakrym** [comment]
Add a test helper method VerifyEventLogEvent() that uses debug log for now. We will switch to something better as soon as we can.

File: test/Common.FunctionalTests/Utilities/Helpers.cs [hunk 1]
```diff
@@ -108,5 +110,13 @@ public static DeploymentParameters GetBaseDeploymentParameters(string site = nul
         private static string GetWebConfigFile(IISDeploymentResult deploymentResult)
             => Path.Combine(deploymentResult.DeploymentResult.ContentRoot, "web.config");
 
-    }
+        public static async Task AssertStarts(IISDeploymentResult deploymentResult, string path)
```
Comment (pakrym) [inline test/Common.FunctionalTests/Utilities/Helpers.cs]: set path to "/HelloWorld" by default

**Comment by mikeharder** [comment]
@jkotalik: I can manually test in a container after this is merged and a hosting bundle is built.

**Comment by jkotalik** [comment]
IMO it's low priority to manually verify this. But feel free to if you think it's worth it.

**Comment by jkotalik** [comment]
>   Total tests: Unknown. Passed: 118. Failed: 0. Skipped: 1.

Appveyor failed to run all iisexpress tests. Maybe due to calling exit(0)?

**Comment by jkotalik** [comment]
https://github.com/aspnet/IISIntegration/issues/1049

**Pull request merged**